src/goto-programs/goto_clean_expr.cpp

```diff
@@ -93,8 +93,6 @@ bool goto_convertt::needs_cleaning(const exprt &expr)
   // g1 = (i == 0)
   // g2 = (i > 10)
   // forall (i : int) (g1 || g2)
-  if(expr.id()==ID_forall || expr.id()==ID_exists)
-    return false;
```

Comment:
The comment above this code needs to be moved, it doesn't actually make sense when these lines are removed.